File: 1st/Jinseon/마지막두원소.py
def solution(num_list):
    answer = num_list
    len_ = len(num_list)
    if num_list[len_-1] > num_list[len_-2]:
        answer.append(num_list[len_-1] - num_list[len_-2])
    else:
        answer.append(num_list[len_-1] * 2)
    return answer

File: 4st/Gau/programmer/마지막 두 원소.java
import java.util.*;

class Solution {
    public List solution(int[] num_list) {
        List<Integer> answer = new ArrayList<>();
        
        int last = num_list[num_list.length - 1];  
        int lastb = num_list[num_list.length - 2];  
        
        for(int i = 0; i < num_list.length; i++){
            answer.add(num_list[i]);
        }
        if(last > lastb){
            answer.add(last - lastb);
        } else {
            answer.add(last * 2);
        }
        return answer;
    }
}